src/python/platforms/android/device.py

```diff
@@ -438,3 +438,20 @@ def setup_host_and_device_forwarder_if_needed():
   for port in ports:
     port_string = 'tcp:%d' % port
     adb.run_command(['reverse', port_string, port_string])
+
+
+def update_build(apk_path, force_update=True):
+  """Prepares the device and updates the build if necessary."""
+  # Prepare device for app install.
+  initialize_device()
+
+  # On Android, we may need to write a command line file. We do this in
+  # advance so that we do not have to write this to the device multiple
+  # times.
+  # TODO(mbarbella): Build code should not depend on fuzzing.
```

Comment:
Please remove this TODO, this is now done. also, can move "from fuzzing " into the module imports at top.

Reply:
Unfortunately we have the same problem in this file. testcase_manager depends on a lot of the rest of the codebase, so importing it from the files with these sorts of helper functions will always be problematic. Not sure what the best way to handle it is, but updated the TODO so it makes sense in this file.